editor/src/utils/vite-hmr-config.ts

```diff
@@ -4,7 +4,7 @@
 // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
 if (import.meta.hot) {
   // this is written without the != null part to make sure Webpack (and Vite) recognizes and tree shakes it
-  import.meta.hot?.decline() // this _should_ be working but does not for some reason
+  import.meta.hot?.invalidate() // this _should_ be working but does not for some reason
```

Comment:
Is it still true that it doesn't work?

Reply:
It very much works, this method is the recommended replacement for `decline` as per the docs: https://vitejs.dev/guide/api-hmr.html#hot-decline